### PR TITLE
Add adaptivePtime to RTCRtpEncodingParameters

### LINKED ISSUE #3: API to allow the UA to adapt ptime dynamically
This was discussed over at webrtc-pc here:
- Issue: https://github.com/w3c/webrtc-pc/issues/2300
- PR: https://github.com/w3c/webrtc-pc/pull/2309

Let's merge an equivalent of that PR in this extension spec. See review status there. Address nits and merge here.

### PR DESCRIPTION
**Pull request by eladalon1983**
Fixes #3.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -156,5 +156,39 @@ <h2>Attributes</h2>
...
+    };</pre>
+    <h2>Dictionary <code><a>RTCRtpEncodingParameters</a></code> Members</h2>
+    <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for="RTCRtpEncodingParameters" class="dictionary-members">
+      <dt><dfn data-idl><code>adaptivePtime</code></dfn> of type <span class="idlMemberType">boolean</span>, defaulting to <code>false</code>.</dt>
+      <dd>
+        <p>Indicates whether this encoding MAY dynamically change
+        the frame length. If the value is <code>true</code>,
+        the user agent MAY use any valid frame length for any of its
+        frames, and MAY change this at any time. Valid values are positive
+        integers divisible by 10. If the <code>maxptime</code> attribute (defined in
```
Comment (fippo) [inline index.html]: multiples of 10ms?

File: index.html [hunk 2]
```diff
@@ -156,5 +156,39 @@ <h2>Attributes</h2>
...
+        [[RFC4566]] Section 6) is specified, that maximum applies.
+        <a href="https://w3c.github.io/webrtc-pc/#dfn-read-only-parameter">Read-only parameter.</a></p>
+        <p class="note">Using a longer frame length reduces the
+        bandwidth consumption due to overhead, but does so at the cost
+        of increased latency. Allowing the user agent to change the
```
Comment (fippo) [inline index.html]: shorter: Changing the frame length dynamically allows the user agent to adapt its bandwidth ...

File: index.html (hunk 1 above)
Comment (eladalon1983) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (eladalon1983) [reply index.html]: Done.

**Pull request merged**